cmd/smokescreen.go

```diff
@@ -85,6 +85,15 @@ func NewConfiguration(args []string, logger *log.Logger) (*smokescreen.Config, e
 			Name:  "egress-acl-file",
 			Usage: "Validate egress traffic against `FILE`",
 		},
+		cli.StringFlag{
+			Name:  "prometheus-endpoint",
+			Usage: "Expose metrics via prometheus on `ENDPOINT`.",
+		},
```

Comment:
Maybe this is just how my brain works with cli args but can we default this to /metrics and add an additional flag to the effect of `metric-type` where you would choose statsd and/or prometheus and then ride the defaults if needed?


As of now the default is always shipping statsd metrics. Might want to provide a toggle to flip those off.

Reply:
@alex-kalinowski, I've pushed a change to add the `--expose-prometheus-metrics` flag to toggle exposing prometheus metrics.

Currently the default isn't shipping statsd metrics? A default flag is set for `statsd-address`, but unless the flag is explicitly passed in, `IsSet("statsd-address")` will return false, and the `SetupStatsd()` method won't be called.

Reply:
I assumed IsSet() returned true for defaults as well.